Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/typings.json

```diff
@@ -1,9 +1,13 @@
 {
-  "dependencies": {},
   "globalDependencies": {
-    "express": "registry:dt/express#4.0.0+20160708185218",
-    "express-serve-static-core": "registry:dt/express-serve-static-core#4.0.0+20160829034835",
+    "body-parser": "registry:dt/body-parser#0.0.0+20160317120654",
+    "cookie-parser": "registry:dt/cookie-parser#1.3.4+20160316155526",
```

Comment:
So were all these other files added here (body-parser and cookie-parser) already present?